app/main.py

```diff
@@ -5,4 +5,4 @@ class Person:
 
 def create_person_list(people: list) -> list:
     # write your code here
-    pass
+    pass
```

Comment:
The `create_person_list` function is currently not implemented. You need to write the logic to create and return a list of `Person` objects based on the input `people` list. Ensure that you follow the task requirements for this implementation.